ui/perfherder/compare/CompareTable.jsx

```diff
@@ -28,24 +39,93 @@ export default class CompareTable extends React.PureComponent {
       displayNumber(percentage),
     )}% ${improvement ? 'better' : 'worse'})`;
 
+  compareBy = key => {
+    const { sortOrderAsc } = this.state;
+    return function(a, b) {
+      if (a[key] <= b[key]) {
+        if (sortOrderAsc) return -1;
+        return 1;
+      }
+      if (a[key] >= b[key]) {
+        if (sortOrderAsc) return 1;
+        return -1;
+      }
+      return 0;
+    };
+  };
+
+  sortBy = (key, data) => {
+    const { sortOrderAsc } = this.state;
+    data.sort(this.compareBy(key));
+    this.setState({ sortOrderAsc: !sortOrderAsc });
+
+    $(
```

Comment:
Changing classes for elements is better to be done directly in render(), something like in this example: https://github.com/mozilla/treeherder/blob/master/ui/perfherder/graphs/TestCard.jsx#L41